### PR TITLE
Branch fix bugs

### PR DESCRIPTION
**Pull request by Darren159**

### CODE REVIEW

File: docs/diagrams/AddRecordSequenceDiagram.puml [hunk 1]
```diff
@@ -49,8 +49,8 @@ deactivate AddressBookParser
 LogicManager -> AddRecordCommand : execute(model)
 activate AddRecordCommand
 
-AddRecordCommand -> Model : setPerson(targetPerson, personWithAddedRecord)
-AddRecordCommand -> Model : updateRecordList(personWithAddedRecord)
+AddRecordCommand -> Model : setPerson(targetPerson, newPerson)
+AddRecordCommand -> Model : updateRecordList(newPerson, index)
```
Comment (hjoneweek) [inline docs/diagrams/AddRecordSequenceDiagram.puml:53]: There isn't index parameter anymore

Comment (Clin-lyx) [inline docs/diagrams/AddRecordSequenceDiagram.puml:53]: No Index parameter for the updateRecordList

**Review by hjoneweek — COMMENTED** [review]
Just a minor thing to change regarding updateRecordList()

**Review by Clin-lyx — COMMENTED** [review]
LG, just fix this

**Review by hjoneweek — APPROVED** [review]
LGTM

**Pull request merged**